github/search.go

```diff
@@ -172,18 +179,52 @@ func (c CodeResult) String() string {
 // GitHub API docs: https://developer.github.com/v3/search/#search-code
 func (s *SearchService) Code(ctx context.Context, query string, opt *SearchOptions) (*CodeSearchResult, *Response, error) {
 	result := new(CodeSearchResult)
-	resp, err := s.search(ctx, "code", query, opt, result)
+	resp, err := s.search(ctx, "code", &searchParameters{Query: query}, opt, result)
+	return result, resp, err
+}
+
+// LabelsSearchResult represents the result of a code search.
+type LabelsSearchResult struct {
+	Total             *int          `json:"total_count,omitempty"`
+	IncompleteResults *bool         `json:"incomplete_results,omitempty"`
+	Labels            []LabelResult `json:"items,omitempty"`
```

Comment:
Two things:
s/Labels/Items/
s/[]LabelResult/[]*LabelResult/

Reply:
Agree with `s/[]LabelResult/[]*LabelResult/`.

However, I don't think `s/Labels/Items/` should be done for the same reason as above. The other `*SearchResult` structs all use renamed Go fields, not `Items`.

Reply:
OK, SGTM.